flash.sh

```diff
@@ -4,27 +4,46 @@ set -eux
 
 artifact="$(basename $1)"
 rust_target_folder="$(readlink -f $(dirname $1)/../..)"
-libtock_target_path="${rust_target_folder}/tab/${PLATFORM}/${artifact}"
-elf_file_name="${libtock_target_path}/cortex-m4.elf"
-tab_file_name="${libtock_target_path}.tab"
-
-mkdir -p "${libtock_target_path}"
-cp "$1" "${elf_file_name}"
-
-elf2tab -n "${artifact}" -o "${tab_file_name}" "${elf_file_name}" --stack 2048 --app-heap 1024 --kernel-heap 1024 --protected-region-size=64
 
 case "${PLATFORM}" in
     "nrf52"|"nrf52840")
         tockloader_flags="--jlink --arch cortex-m4 --board nrf52dk --jtag-device nrf52"
-        ;;
+	binary_name=cortex-m4.elf
```

Comment:
I like this solution!

Sorry for being pedantic but are the lines wobbling?

Reply:
Sorry, I'm not sure what you are saying with wobbling. Do you mean something something in the style?

Reply:
Yes, e.g. line 20 is indented by more characters than line 21 when viewing it in the browser.

Reply:
Ah yes, I was using tabs instead of spaces. I fixed it to be consistent.